.github/workflows/auto_branching.yml

```diff
@@ -138,7 +138,7 @@ jobs:
       - name: Commit changes
         run: |
           git config --local user.email Satellite-QE.satqe.com && git config --local user.name Satellite-QE
-          git add pyproject.toml ./tests/foreman ./robottelo/* ./requirements.txt ./.github/*
+          git add pyproject.toml ./tests/foreman ./robottelo/* ./requirements.txt ./.github/
```

Comment:
should the same be applied to `robotello/*`? or, alternatively, explicitly add `./robottelo/constants/__init__.py`?

Reply:
Just FTR, we do update constants:
https://github.com/SatelliteQE/robottelo/pull/19409/files#diff-6725dcbd683c84f65abd7793249c3a251ce7ba21899ba735e61c994161191402R124-R130

Reply:
ah, `git add ./robottelo/` should take care of that update as well